files/en-us/web/http/headers/content-digest/index.html

```diff
@@ -0,0 +1,28 @@
+---
+title: Content-Digest
+slug: Web/HTTP/Headers/Content-Digest
+page-type: http-header
+browser-compat: http.headers.Content-Digest
+spec-urls: https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-digest-headers-12
+status:
+  - experimental
+---
+
+{{HTTPSidebar}}{{SeeCompatTable}}
+
+The **`Content-Digest`** response or request header provides a {{Glossary("digest")}} of the _actual message content_, the preimage of computing the _selected representation_ (e.&hairsp;g. [Range Requests](/en-US/docs/Web/HTTP/Range_requests) or [Content Encoding](/en-US/docs/Web/HTTP/Content_negotiation)).  
```

Comment:
Avoid "e.g.". You'll see it a bit, but we tend to expand out to For example because screen readers do a poor job with these kinds of abbreviation. We also use highlighting sparingly.

```suggestion
The **`Content-Digest`** response or request header provides a {{Glossary("digest")}} of the actual message content, the preimage of computing the _selected representation_.
For example, [Range Requests](/en-US/docs/Web/HTTP/Range_requests) or [Content Encoding](/en-US/docs/Web/HTTP/Content_negotiation).  
```